alembic/versions/52693750d371_removing_player_rating_table.py

```diff
@@ -0,0 +1,48 @@
+"""Removing player rating table

```

Comment:
Will it be possible to override the rank value of a play in the database? We have several players who play on alternate accounts because they have more champs/skins than a higher ranked main account.

Reply:
To remediate this issue just have them verify as their main account, and they can play on whatever account they choose to in-game

Also you can make a quick code change, where you make a new table and have like a !setRank @user {rank} and upload to table and check to see if player exists in table and use that value instead or something like that